style.css

```diff
@@ -55,7 +57,7 @@ body {
   /* animation changes */
 }
 
-.headr .navbar a {
+.headr .navbar .links {
   font-style: normal;
   font-weight: 500;
   font-size: 15px;
```

Comment:
- [ ] There is a side-scrolling in the mobile version and mobile menu to get to the hamburger icon and the close icon, So please fix that.

- [ ] Please make the background of the mobile nav menu blurry to match the given design.
[hint] You can do that using [backdrop-filter](https://www.w3schools.com/cssref/css3_pr_backdrop-filter.php) CSS property.

![chrome-capture-2023-0-31](https://user-images.githubusercontent.com/68796232/215628094-b1003d39-0528-493c-b208-7fe22f75db83.gif)

Reply:
Fixed.